tests/ui/single_match.rs

```diff
@@ -244,3 +244,24 @@ fn main() {
         _ => 0,
     };
 }
+
+fn issue_10808(bar: Option<i32>) {
+    match bar {
+        Some(v) => unsafe {
+            let r = &v as *const i32;
+            println!("{}", *r);
+        },
+        _ => {},
+    }
+
+    match bar {
+        Some(v) => {
+            // this comment prevents rustfmt from collapsing the block
```

Comment:
Using `#[rustfmt::skip]` would be better.

Reply:
Thanks @Jarcho, fixed